Src/WindowsServer/WindowsServer.Shared/Implementation/AzureHeartbeatProperties.cs

```diff
@@ -0,0 +1,116 @@
+﻿namespace Microsoft.ApplicationInsights.WindowsServer.Implementation
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.ApplicationInsights.Extensibility.Implementation.Tracing;
+
+    internal class AzureHeartbeatProperties
```

Comment:
Please rename to indicate the true purpose of this class. Its not just a list of properties, so current name does not do full justice

Reply:
Agreed. Although a tad verbose, I have changed this to `AzureComputeMetadataHeartbeatPropertyProvider`.